CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 
 -Features for next release
 
+## 2.26.0
```

Comment:
```suggestion
```

I think both can be removed (the version and the "-Features for next release"). one of the older PR's: https://github.com/aws-amplify/aws-sdk-ios/pull/3709/files